docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -0,0 +1,372 @@
+# Defining placement rules in SQL
+
+- Author(s):     [djshow832](https://github.com/djshow832) (Ming Zhang)
+- Last updated:  2020-06-24
+- Discussion at: https://docs.google.com/document/d/18Kdhi90dv33muF9k_VAIccNLeGf-DdQyUc8JlWF9Gok
```

Comment:
It seems that this link is private, is it OK?

Reply:
It's public now, to any one who knows this link.

Reply:
I think there existed some internal links in this document, maybe we should do some desensitization work.